From akpm@linux-foundation.org Fri Mar 28 14:23:24 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Fri, 28 Mar 2008 14:22:52 -0700
Subject: PM: Remove legacy PM (fix)
To: greg@kroah.com
Cc: akpm@linux-foundation.org, randy.dunlap@oracle.com, pavel@suse.cz, rjw@sisk.pl
Message-ID: <200803282122.m2SLMqvo010807@imap1.linux-foundation.org>


From: Randy Dunlap <randy.dunlap@oracle.com>

Source file was removed.  Need to remove docbook reference also.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Pavel Machek <pavel@suse.cz>
Cc: Rafael J. Wysocki <rjw@sisk.pl>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/DocBook/kernel-api.tmpl |    5 -----
 1 file changed, 5 deletions(-)

--- a/Documentation/DocBook/kernel-api.tmpl
+++ b/Documentation/DocBook/kernel-api.tmpl
@@ -297,11 +297,6 @@ X!Earch/x86/kernel/mca_32.c
 !Ikernel/acct.c
   </chapter>
 
-  <chapter id="pmfuncs">
-     <title>Power Management</title>
-!Ekernel/power/pm.c
-  </chapter>
-
   <chapter id="devdrivers">
      <title>Device drivers infrastructure</title>
      <sect1><title>Device Drivers Base</title>
